recipes/supportdata/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "supportdata" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 606346ee39642a8395814b3a55bc9f9b8273414cc02311b45e4a44307272a51d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - filelock >=2.0
+    - pip
+    - python
+    - six >=1.5.2
+  run:
+    - filelock >=2.0
+    - python
+    - six >=1.5.2
+
+test:
+  imports:
+    - supportdata
+
+about:
+  home: "https://github.com/castelao/supportdata"
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.rst
```

The license file is already being packaged on ``pypi``, so you do not need to add it manually 
```suggestion
  license_file: LICENSE
```

You can also remove the license file in this PR